approach.md

```diff
@@ -3,8 +3,28 @@ layout: page
 title: Our Approach
 ---
 
-_Distributed Press_ is a proposal to build an open-source publishing tool for the distributed web. Our approach is to engage communities actively researching the many issues we face in content publishing today to create a tool that prioritizes content authors. In particular, we hope to learn from independent media that are practicing alternative publishing models, to investigate if and how the distributed web may augment this ecosystem.
+_Distributed Press_ is an initiative to build an open-source publishing tool for the distributed web.
 
-Beginning this year, we interviewed activists and journalists who combat misinformation and censorship, founders of progressive platforms like Popula and Are.na, builders of open protocols from the IndieWeb and DWeb movement. Some of these interviews are published on [GitHub](https://github.com/benhylau/distributed.press/blob/master/interviews.md). As our project develops, we will continue to collaborate with these communities and many other neighbors.
+Our approach begins with engaging communities actively researching the wide-ranging issues we face in content publishing today, learning from independent media that are practicing alternative models of publishing and governance, as well as progressive technology standards and experimental platforms.
```

Comment:
Thanks @uditvira do you know about this feature :)

```suggestion
Our approach involves engaging communities, learning from adjacent media projects, and experimenting with novel technologies and standards. This will help us address a wide range of issues in publishing by establishing a thriving community sustained through good governance and an equitable revenue model.
```

Reply:
Ah yes, I knew you could do that but couldn't locate the right button. But found it now, thanks!